packages/remix-dev/vite/build.ts

```diff
@@ -284,7 +246,15 @@ export async function build(
   await preloadViteEsm();
 
   let viteConfig = await resolveViteConfig({ configFile, mode, root });
-  let ctx = await extractRemixPluginContext(viteConfig);
+
+  // eslint-disable-next-line prefer-let/prefer-let -- Improve type narrowing
+  const ctx = await extractRemixPluginContext(viteConfig);
+
+  if (!ctx) {
+    console.error(colors.red("Remix Vite plugin not found in Vite config"));
+    process.exit(1);
+  }
```

Comment:
This error used to be in `extractRemixPluginContext` but since it's now used in contexts where it's not guaranteed that the consumer is using Vite, `extractRemixPluginContext` now returns `undefined` if Vite and/or the Remix Vite plugin aren't found, so we handle the error here now instead.